ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -152,7 +165,9 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
 
   const { isLoading } = useLoadingEffect(async () => {
     const results = await Api.executeSearchExpression(portal.shortcode,
-      study.shortcode, currentEnv.environmentName, searchExpression)
+      study.shortcode,
+      currentEnv.environmentName,
+      concatSearchExpressions([searchExpression, 'include({user.lastLogin})']))
```

Comment:
this feels a bit strange, that we have to auto-add `user.lastLogin`, but we don't have to auto-add `profile`.  Maybe that's fine, since rule evaluations often use profile info, but will pretty much never use participantUser info

Reply:
That was my thought, but I'm happy to require `include`ing profile stuff now that it is possible to do so